### DISCUSSION TITLE
Plugin Common Options File

### DISCUSSION THREAD

**Discussion by JustinianErdmier** [JetBrains/rider-efcore#118] [Q&A] [answered]
Is there a sample for with all the defaults for the plugin's efCoreCommonOptions.xml?

It created the file, storing my preference for the migrations and startup projects. However, I'd like to add an entry for my Migrations folder so I don't have to manually set it each time. It's heavily nested.

Much appreciation.

**Comment by seclerp** [comment, marked as answer]
Hi Justin, currently it's not possible. This file is not dynamic right now and it can't be mapped to a whole set of fields in dialogs.

However, I'm spending a lot of time right now on refactoring, including ability to implement more dialogs options persistence later. You could track it here: #54.

After the above task will be done - you could help in implementing a portion of your favorites' dialog selected options saving!

Anyway in the next major release all dialogs will be more friendly in such a sense.
